crates/encoding/src/glyph_cache.rs

```diff
@@ -49,18 +51,16 @@ impl GlyphCache {
             };
             encoding_cache.encode_fill_style(fill);
             let mut path = encoding_cache.encode_path(true);
+            let outline = outlines.get(GlyphId::new(key.glyph_id as u16))?;
+            let draw_settings = skrifa::outline::DrawSettings::unhinted(size, coords);
```

Comment:
```suggestion
            // FIXME: Re-add hinting when skrifa supports it
            let draw_settings = skrifa::outline::DrawSettings::unhinted(size, coords);
```

Reply:
Sure, a little reminder won't hurt. Also added a link to the tracking issue.